DOCKER.md

```diff
@@ -53,15 +54,18 @@ For example, "What is a maker schedule?"
 
 ### 4. Run the application
 
-Execute the run script from the root directory, specifying your configuration as `config/pg-search`, and selecting the `default` profile from your secrets file:
+Execute the run script from the root Vectara Answer folder, specifying your configuration as `config/pg-search`, and selecting the `default` profile from your secrets file:
 
 ```sh
 bash docker/run.sh config/pg-search default
 ```
 
-The application executes inside a [Docker container](https://www.docker.com/resources/what-container/), which ensures a clean build and avoids any issues with existing environments and installed packages.
-
-When the container is set up, the `run.sh` launch script will open up a browser at `localhost:80`.
+Thr `run.sh` script performs the following steps:
+* runs the `prepare_config.py` python script which creates the `.env` file using the `config.yaml` and `secrets.toml` files
+* Creates the [Docker container](https://www.docker.com/resources/what-container/) and runs this container in your local Docker Desktop instance. 
+The `queries.json` file is loaded dynamically as a Docker volume file.
```

Comment:
Seems like this should be a step, too, right?

Reply:
Not a step. It's more a note of how queries.json is handled during execution of step 2.

Reply:
Oh, I see. In that case I suggest deleting the newline so it's clearer that it's part of that step:

```md
* Creates the [Docker container](https://www.docker.com/resources/what-container/) and runs this container in your local Docker Desktop instance. The `queries.json` file is loaded dynamically as a Docker volume file.
```